cmd/peer/peer.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2020 Authors of Hubble
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package peer
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	peerpb "github.com/cilium/cilium/api/v1/peer"
+	"github.com/cilium/hubble/pkg/defaults"
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+var (
+	serverURL     string
+	serverTimeout time.Duration
+)
+
+// New creates a new hidden peer command.
+func New() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "peer",
+		Aliases: []string{"peers"},
```

Comment:
the reverse reads better: peers by default, peer alias

`hubble peers list` rather than `hubble peer list` and would generate better help page probably

Reply:
It does. Updated.